swift_browser_ui_frontend/src/common/lang.js

```diff
@@ -526,24 +530,28 @@ let default_translations = {
                 "epäonnistunut salausohjelman lataus tai tilan loppuminen. " +
                 "Päivitä sivu ja kokeile uudelleen.",
         files: "Tiedostot",
-        ephemeral: "Tilapäinen yksityinen avain",
+        ephemeral: "Käyttää omaa yksityistä avainta",
+        multipleReceivers: "Lisää muita vastaanottimien julkisia avaimia",
```

Comment:
~vastaanottimien~ -> vastaanottajien
Google Translate apparently doesn't get the context difference between a receiver as a person and radio receiver